src/impl.h

```diff
@@ -324,9 +324,7 @@ struct Manifold::Impl {
 
   // face_op.cpp
   void Face2Tri(const Vec<int>& faceEdge, const Vec<TriRef>& halfedgeRef);
-  PolygonsIdx Face2Polygons(VecView<Halfedge>::IterC start,
-                            VecView<Halfedge>::IterC end,
-                            mat2x3 projection) const;
```

Comment:
I split Face2Polygons into two non-member functions: AssembleHalfedges and ProjectPolygons.